llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -5553,6 +5553,14 @@ static SDValue getSVEPredicateBitCast(EVT VT, SDValue Op, SelectionDAG &DAG) {
   if (InVT == VT)
     return Op;
 
+  // Look through casts to <n x 16 x i1> when their input has more lanes than
```

Comment:
Perhaps better written as `casts to <vscale x 16 x i1>` in case it's confusing?

Reply:
I knew something was odd when I was writing it but couldn't figure it out :) I'll change it to `MVT::nxv16i1` if that's ok?

Reply:
Done, plus removed another redundant reference to `<n x 16 x i1>`.